package.json

```diff
@@ -665,6 +665,15 @@
           "description": "Path to run neovim executable. For example, /usr/bin/nvim, or  C:\\Program Files\\Neovim\\bin\\nvim.exe",
           "default": "nvim"
         },
+        "vim.enableVimrc": {
```

Comment:
Can we move the configs to under something like this:

`vim.vimrc.enable`
`vim.vimrc.path`